config/locales/en.yml

```
@@ -350,6 +350,14 @@ en:
       collaboration/access/read_only: "readonly"
     errors:
       models:
+        investigation/enquiry:
+          attributes:
+            date_received:
+              blank: Enter the date the enquiry was received
+              must_be_real: Date received of enquiry must be a real date
+              incomplete: Date received of enquiry must include a %{missing_date_parts}
+              in_future: Date received of enquiry must be today or in the past
```

Think these need a bit of re-wording...

Suggestion:

```suggestion
              must_be_real: Date enquiry received must be a real date
              incomplete: Date enquiry received must include a %{missing_date_parts}
              in_future: Date enquiry received must be today or in the past
```

---

cheers! yes does read much better indeed.